tools/Dashboard/DevHome.Dashboard/Views/AddWidgetDialog.xaml.cs

```diff
@@ -74,15 +74,24 @@ private async Task FillAvailableWidgetsAsync()
             return;
         }
 
-        var providerDefs = _hostingService.GetWidgetCatalog()!.GetProviderDefinitions();
-        var widgetDefs = _hostingService.GetWidgetCatalog()!.GetWidgetDefinitions();
+        var providerDefinitions = _hostingService.GetWidgetCatalog()!.GetProviderDefinitions();

```

Comment:
Why not put the `.OrderBy()` here?

Reply:
o I see.  Checking for null.  NVM.

Reply:
I was thinking to check for null first, but I suppose GetProviderDefinitions can never return null because Dev Home will always be a widget provider.